gpu_memory_management/test/test.py

```diff
@@ -0,0 +1,74 @@
+import pytest
```

Comment:
Just as hint. Pytest executes all files, stating with the prefix `test_` automatically in the folder. So if you rename your `test.py` to `test_<something>.py` you can execute `pytest` in the build folder and pytest will execute all tests in the file automatically.